packages/bruno/PKGBUILD

```diff
@@ -0,0 +1,100 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+# Initial PKGBUILD from AUR.
+# Old Maintainer: Vianney Bouchaud <aur dot vianney at bouchaud dot org>
+
+pkgname=bruno
+pkgver=1.34.2
```

Comment:
For now, we mostly do git packages instead of release ones, could you adapt from the AUR?

Reply:
Fixed